src/com/hishamfactory/Company.java

```diff
@@ -4,16 +4,19 @@
 
 public class Company {
     private String name;
+    private String uuid;
```

Comment:
after company object created don't have UUID,fix it.

Reply:
fixed it